common/djangoapps/track/shim.py

```diff
@@ -119,15 +121,40 @@ def __call__(self, event):
 
             del payload['module_id']
 
+        if 'context' not in event:
+            return
+
+        context = event['context']
+
+        #For the iOS build that is returning a +30 for back skip 30
+        if context['application']['version'] == "1.0.02" and \
+                        context['application']['name'] == "edx.mobileapp.iOS":
+            if 'requested_skip_interval' in payload and 'seek_type' in payload:
+                if payload['requested_skip_interval'] == 30 and \
+                        payload['seek_type'] == "skip":
+                    payload['requested_skip_interval'] = -30
+
+        #For the Android build that isn't distinguishing between skip and seek
+        if 'requested_skip_interval' in payload:
+            if payload['requested_skip_interval'] != -30:
```

Comment:
can we compare with both -30 and +30?  or take the absolute value of the requested_skip_interval?
this way, this code will handle both rewind and forward features.
or, just in case the android build ever sends +30 by mistake again (argh!), at least we won't change the seek_type.  :)